dev.py

```diff
@@ -409,7 +409,12 @@ def setup_build(args, env):
     """
     cmd = ["meson", "setup", args.build_dir, "--prefix", PATH_INSTALLED]
     build_dir = Path(args.build_dir)
-    if os.path.exists(build_dir / 'build.ninja'):
+    if (os.path.exists(build_dir) and not os.path.exists(build_dir
+                                                         / 'meson-info')):
```

Comment:
Sorry to be nit-picky - but given this is already a Path, why not use the methods, like `build_dir.exists()`?

```suggestion
    if build_dir.exists() and not (build_dir / 'meson-info').exists():
```